backend/src/schema/resolvers/users.js

```diff
@@ -23,6 +23,21 @@ export const getMutedUsers = async context => {
   return mutedUsers
 }
 
+export const getBlockedUsers = async context => {
+  const { neode } = context
+  const userModel = neode.model('User')
+  let blockedUsers = neode
+    .query()
+    .match('user', userModel)
+    .where('user.id', context.user.id)
+    .relationship(userModel.relationships().get('blocked'))
+    .to('blocked', userModel)
+    .return('blocked')
+  blockedUsers = await blockedUsers.execute()
+  blockedUsers = blockedUsers.records.map(r => r.get('blocked').properties)
```

Comment:
Note that this will be imperformant on nested objects as we're not using `neo4j-graphql-js` to construct the query.

Reply:
it's copied from what we have in production, but noted...